src/plugins/intel_cpu/src/emitters/README.md

```diff
@@ -0,0 +1,116 @@
+
+
+## Development
+
+JIT emitters are inherited from `jit_emitter` base class. The base class implementation depends on architecture:
+ * X64: [jit_emitter.hpp](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/emitters/plugin/x64/jit_emitter.hpp)
+ * ARM64: [jit_emitter.hpp](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_emitter.hpp)
+
+### Class diagram
+JIT emitters should be inheritted from `jit_emitter` base class and it's usage should be added in `jit_uni_eltwise_generic` ARM64 kernel. 
+```mermaid
+classDiagram
+jit_uni_eltwise_generic o-- jit_operation_emitter
+jit_operation_emitter --|> jit_emitter
+jit_emitter --|> Emitter
+```
+
+### Sequential diagram
+
+```mermaid
+sequenceDiagram
+    participant jit_uni_eltwise_generic
+    participant jit_operation_emitter
+    participant jit_emitter
+    
+    jit_uni_eltwise_generic->>jit_uni_eltwise_generic: preamble
+
+    Note left of jit_operation_emitter: preparation
+    jit_uni_eltwise_generic->>jit_operation_emitter: get_supported_precisions
+    jit_operation_emitter->>jit_uni_eltwise_generic: 
+
+    jit_uni_eltwise_generic->>jit_operation_emitter: constructor
+    jit_operation_emitter->>jit_emitter: prepare_table
+    jit_emitter->>jit_operation_emitter: 
+    jit_operation_emitter->>jit_operation_emitter: register_table_entries
+    jit_operation_emitter->>jit_uni_eltwise_generic: 
+    jit_operation_emitter->>jit_operation_emitter: create dependent emitter instance
+    jit_operation_emitter->>jit_uni_eltwise_generic: 
+
+    jit_uni_eltwise_generic->>jit_operation_emitter: get_inputs_count
+    jit_operation_emitter->>jit_uni_eltwise_generic: 
+
+    jit_uni_eltwise_generic->>jit_operation_emitter: get_aux_vecs_count
+    jit_operation_emitter->>jit_uni_eltwise_generic: 
+
+    jit_uni_eltwise_generic->>jit_operation_emitter: get_aux_gprs_count
+    jit_operation_emitter->>jit_uni_eltwise_generic: 
+
+    Note left of jit_operation_emitter: code emittion
+    jit_uni_eltwise_generic->>jit_emitter: emit_code
+    jit_emitter->>jit_emitter: emitter_preamble
+    jit_emitter->>jit_operation_emitter: emit_impl
+    jit_operation_emitter->>jit_operation_emitter: emit_isa
+    jit_operation_emitter->>jit_emitter: 
+    jit_emitter->>jit_emitter: emitter_postamble
+    jit_emitter->>jit_uni_eltwise_generic: 
+
+    Note left of jit_operation_emitter: data emittion
+    jit_uni_eltwise_generic->>jit_operation_emitter: emit_data
+    jit_operation_emitter->>jit_emitter: emit_data
+    jit_emitter->>jit_operation_emitter: 
+    jit_operation_emitter->>jit_operation_emitter: dependent emitter: emit_data 
+    jit_operation_emitter->>jit_uni_eltwise_generic: 
+    
+    jit_uni_eltwise_generic->>jit_uni_eltwise_generic: postamle
+```
+
+## Debugging
```

Comment:
I would also propose to add info about our register printing capabilities https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/emitters/plugin/x64/utils.hpp#L54. They are implemented only for x64 for now, but definetelly can be extended for aarch64 (btw another good task for GFI scope),

Reply:
added, thanks